gating/gate/run

```diff
@@ -26,4 +26,25 @@
 # trigger.
 # RE_JOB_REPO_NAME: The project's GitHub repository which gets cloned in the
 # job. For example, for the repository found at
-# https://github.com/rcbops/rpc-gating, the name is rpc-gating.
\ No newline at end of file
+# https://github.com/rcbops/rpc-gating, the name is rpc-gating.
+
+set -eux
+set -o pipefail
+
+cd /opt/magnanimous-turbo-chainsaw/scripts/
+. setup.sh
+
+# Ensure the deployment workspace3 is loaded and ready to run workloads
+. setup-workspace.sh
+
+# Probe the environment for any host that is unreadchable
+cd /opt/magnanimous-turbo-chainsaw/playbooks/
+ansible-playbook probe-systems.yml
+
```

Comment:
before running these scripts in test we should deactivate the workspace so that we're ensuring 1) we are not running into issues due to running a venv in a venv, and 2) we validate the scripts work as stand alone resources.

Reply:
That should probably be documented in the Readme. These steps are pulled directly from there. Are you suggesting the deactivate happen after setup-workspace, but before moving into playbooks, and executing the probe-systems playbook?

Reply:
I'm assuming before the run-tests script.